src/app/i18n/locales/all-locales.d.ts

```diff
@@ -228,6 +228,7 @@ interface AllLocalesType {
 
     delivery: string[];
     btn_to_basket: string;
+    EmptyCatalogMsgs: string[];
```

Comment:
Please, rename the variable by example empty_catalog

Reply:
done